engine/src/main/java/org/terasology/persistence/internal/SaveTransaction.java

```diff
@@ -91,13 +95,19 @@
     private final StoragePathProvider storagePathProvider;
     private final SaveTransactionHelper saveTransactionHelper;
 
+    //Record and Replay
+    private RecordAndReplaySerializer recordAndReplaySerializer;
+    private RecordAndReplayUtils recordAndReplayUtils;
 
-    public SaveTransaction(EngineEntityManager privateEntityManager, EntitySetDeltaRecorder deltaToSave,
+
+    SaveTransaction(EngineEntityManager privateEntityManager, EntitySetDeltaRecorder deltaToSave,
```

Comment:
Access change - *not* an API violation in this case since internal, but dubious it is needed/intentional?